cid/common.py

```diff
@@ -670,8 +671,11 @@ def _share(self, dashboard_id, **kwargs):
                 })
 
             logger.info(f'Sharing dashboard {dashboard.name} ({dashboard.id})')
-            self.qs.update_dashboard_permissions(DashboardId=dashboard.id, **dashboard_params)
-            logger.info(f'Shared dashboard {dashboard.name} ({dashboard.id})')
+            try:
```

Comment:
This should be handled inside the `update*` function

Reply:
Well, i would say that the calling function generally knows the context - if AccessDeniedException can be safely ignored or not.